src/state/utils.ts

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright © 2021-2023 Technology Matters
```

Comment:
```suggestion
 * Copyright © 2023 Technology Matters
```

Reply:
question: `utils.ts` is moved from `utils.js`, same as with `store` and several other files in this PR, just git wasn't smart enough to realize that. i'm guessing i should not update this copyright since most of the code is not from this PR?

Reply:
Use 2023 for newly created-files.

Reply:
i am still confused, should i also then use 2023 for `store.ts`, `accountSlice.ts`, etc? i think some of those files haven't change contents at all other than the extension